3.2/itop_design.xsd

```diff
@@ -2614,47 +2629,73 @@
     </xs:complexType>
 
-    <xs:simpleType name="MenuEnableActionEnumeration">
-        <xs:restriction base="xs:string">
-            <xs:enumeration value="UR_ACTION_READ" />
-            <xs:enumeration value="UR_ACTION_MODIFY" />
-            <xs:enumeration value="UR_ACTION_DELETE" />
-            <xs:enumeration value="UR_ACTION_BULK_READ" />
-            <xs:enumeration value="UR_ACTION_BULK_MODIFY" />
-            <xs:enumeration value="UR_ACTION_BULK_DELETE" />
-        </xs:restriction>
-    </xs:simpleType>
+    <xs:complexType name="MenuEnableActionEnumeration">
+        <xs:simpleContent>
+            <xs:restriction base="alteredString">
+                <xs:enumeration value="UR_ACTION_READ" />
+                <xs:enumeration value="UR_ACTION_MODIFY" />
+                <xs:enumeration value="UR_ACTION_DELETE" />
+                <xs:enumeration value="UR_ACTION_BULK_READ" />
+                <xs:enumeration value="UR_ACTION_BULK_MODIFY" />
+                <xs:enumeration value="UR_ACTION_BULK_DELETE" />
+            </xs:restriction>
+        </xs:simpleContent>
+    </xs:complexType>
 
-    <xs:simpleType name="MenuEnablePermissionEnumeration">
-        <xs:restriction base="xs:string">
-            <xs:enumeration value="UR_ALLOWED_NO" />
-            <xs:enumeration value="UR_ALLOWED_YES" />
-        </xs:restriction>
-    </xs:simpleType>
+    <xs:complexType name="MenuEnablePermissionEnumeration">
+        <xs:simpleContent>
+            <xs:restriction base="alteredString">
+                <xs:enumeration value="UR_ALLOWED_NO" />
+                <xs:enumeration value="UR_ALLOWED_YES" />
+            </xs:restriction>
+        </xs:simpleContent>
+    </xs:complexType>
 
-    <xs:simpleType name="zeroOrOneEnumeration">
-        <xs:restriction base="xs:unsignedByte">
-            <xs:enumeration value="0" />
-            <xs:enumeration value="1" />
-        </xs:restriction>
+    <xs:complexType name="zeroOrOneEnumeration">
+        <xs:simpleContent>
+            <xs:restriction base="alteredString">
+                <xs:enumeration value="0" />
+                <xs:enumeration value="1" />
+            </xs:restriction>
+        </xs:simpleContent>
+    </xs:complexType>
+
+    <xs:simpleType name="MenuAutoReloadEnumeration">
+        <xs:union memberTypes="xs:nonNegativeInteger">
+            <xs:simpleType>
+                <xs:restriction base="xs:string">
+                    <xs:enumeration value="none" />
+                    <xs:enumeration value="standard" />
+                    <xs:enumeration value="fast" />
+                </xs:restriction>
+            </xs:simpleType>
+        </xs:union>
     </xs:simpleType>
+    <xs:complexType name="alteredMenuAutoReloadEnumeration">
+        <xs:simpleContent>
+            <xs:extension base="MenuAutoReloadEnumeration">
+                <xs:attributeGroup ref="alteredNode" />
+            </xs:extension>
+        </xs:simpleContent>
+    </xs:complexType>
```

Comment:
I would have preferred these to be combined into 1 type, but couldn't find it. Do you have an idea?